SmartDeviceLink/SDLManagerDelegate.h

```diff
@@ -53,9 +53,10 @@ NS_ASSUME_NONNULL_BEGIN
  * Called when the lifecycle manager detected a language mismatch. In case of a language mismatch the manager should change the apps registration by updating the lifecycle configuration to the specified language. If the app can support the specified language it should return an Object of SDLLifecycleConfigurationUpdate, otherwise it should return nil to indicate that the language is not supported.
  *
  * @param language The language of the connected head unit the manager is trying to update the configuration.
+ * @param hmiLanguage The language of the connected head unit the manager is trying to update the configuration.
```

Comment:
This does not properly describe the parameter, it just copies the above.

Reply:
We have changed the description of the parameters